### PR TITLE
[DC-1847] Fix broken achilles integration tests

### PR DESCRIPTION
**Pull request by mark-velez**
By loading csv files from a test bucket

### CODE REVIEW

File: tests/test_util.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 import gcs_utils
 import resources
 
+TEST_RESOURCES_BUCKET = 'aou-res-curation-test-resources'
```
Comment (ksdkalluri) [inline tests/test_util.py:16]: @mark-velez can we store the bucket name in the environment variables? or is it okay to be here?

Comment (mark-velez) [reply tests/test_util.py:16]: Yes, I discussed this with Lauren- this is not great, but it is no worse than the current approach of hardcoding a file_id. I'm reluctant to add another environment variable. Let me know if you can think of any easy alternatives.

Comment (ksdkalluri) [reply tests/test_util.py:16]: @mark-velez  we have project_id already in envireonment_variables, we can leverage it '{project_id}-resources` . let me know your thoughts.

**Comment by mark-velez** [comment]
@ksdkalluri does this approach to getting the bucket look ok? (Excuse the failed builds I added while trying to figure out my lint setup.)

File: tests/test_util.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 import gcs_utils
 import resources
 
+RESOURCES_BUCKET_FMT = '{project_id}-resources'
```
Comment (ksdkalluri) [inline tests/test_util.py:16]: @mark-velez project_id is not yet available yet here.

File: tests/test_util.py [hunk 3]
```diff
@@ -281,33 +239,22 @@ def write_cloud_fp(bucket, name, fp):
     return gcs_utils.upload_object(bucket, name, fp)
 
 
-def populate_achilles(hpo_bucket, hpo_id=FAKE_HPO_ID, include_heel=True):
+def populate_achilles(hpo_id=FAKE_HPO_ID, include_heel=True):
     from validation import achilles, achilles_heel
     import app_identity
 
     app_id = app_identity.get_application_id()
```
Comment (ksdkalluri) [inline tests/test_util.py:246]: @mark-velez this app_id can be used in place of `project_id`.

**Review by ksdkalluri — CHANGES_REQUESTED** [review]

File: tests/test_util.py (hunk 2 above)
Comment (mark-velez) [reply tests/test_util.py:16]: This is a format string where `project_id` is a named index..?

File: tests/test_util.py (hunk 3 above)
Comment (mark-velez) [reply tests/test_util.py:246]: That's what is happening in the line below..?

Comment (ksdkalluri) [reply tests/test_util.py:246]: yes, my bad thats' true. All the tests were passing. Thanks for updating this.

**Review by ksdkalluri — APPROVED** [review]

**Pull request merged**